.github/workflows/pull-request.yml

```diff
@@ -109,7 +109,7 @@ jobs:
         uses: actions/setup-go@v4
         with:
           # NOTE: Keep the version in sync with Go toolchain in WORKSPACE.
-          go-version: '1.20.10'
+          go-version: '1.21'
```

Comment:
golang 1.21 is scheduled tentatively to RHEL 9.4, we aim to release forklift 2.6 before that